com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/cf/CloudControllerClientProvider.java

```diff
@@ -81,31 +74,31 @@ private OAuth2AccessToken getValidToken(String userName, String org, String spac
         return token;
     }
 
-    private Pair<CloudControllerClient, TokenProvider> retrieveClientForToken(String userName, String org, String space, String processId) {
+    private CloudControllerClient retrieveClientForToken(String userName, String org, String space, String processId) {
```

Comment:
maybe rename the method as the "ForToken" part is very misleading now that we don't return any sort of Token/TokenProvider.